### ISSUE TITLE
Assertion failure when subscribing to new feed

### ISSUE THREAD

**Issue by josh64x2** [ViennaRSS/vienna-rss#670]
**Steps to reproduce:**

1. Open Vienna with new database
2. Add new subscription `http://boingboing.net`

```
2016-06-07 09:31:23.468 Vienna[12207:1426651] *** Assertion failure in -[AppController stopProgressIndicator], /Users/josh/src/vienna-rss/src/AppController.m:3914
2016-06-07 09:31:23.470 Vienna[12207:1426651] An uncaught exception was raised
2016-06-07 09:31:23.470 Vienna[12207:1426651] Called stopProgressIndicator without a matching startProgressIndicator
```

**Comment by barijaona** [comment]
Could not reproduce this (using my bugfixes branch)

**Comment by josh64x2** [comment]
Try turning off fetch new subscriptions on start, might only happen if the progress spinner hasn't been started yet.

**Comment by josh64x2** [comment]
fixed in pr #668
